components/inputs/docs/input-radio.md

```diff
@@ -28,127 +23,87 @@ As a result, we have to apply styles to native radio inputs.
 <!-- docs: end dos -->
 
 <!-- docs: start donts -->
-* Don’t use two radio buttons if a single checkbox works better
+* Don’t use two radio inputs if a single checkbox works better
 * Don’t use for triggering an immediate action. Notable exceptions are forms that autosave with clear indication and as a trigger for progressive disclosure on traditional forms, so long as users are made aware that new options have been made available
```

Comment:
This one is actually missing a period at the end, since it's multiple sentences 😬